Simplenote/src/test/java/com/automattic/simplenote/viewmodels/AddTagViewModelTest.kt

```diff
@@ -0,0 +1,82 @@
+package com.automattic.simplenote.viewmodels
+
+import androidx.arch.core.executor.testing.InstantTaskExecutorRule
+import com.automattic.simplenote.R
+import com.automattic.simplenote.repositories.FakeTagsRepository
+import com.automattic.simplenote.utils.getLocalRandomStringOfLen
+import org.junit.Assert.assertEquals
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+
+
+class AddActivityViewModelTest {
+    @get:Rule val rule = InstantTaskExecutorRule()
+
+    private lateinit var viewModel: AddTagViewModel
+    private val fakeTagsRepository = FakeTagsRepository()
+
+    @Before
+    fun setup() {
+        fakeTagsRepository.clear()
+        fakeTagsRepository.failAtSave = false
+
+        viewModel = AddTagViewModel(fakeTagsRepository)
+    }
+
+    @Test
+    fun validateEmptyTag() {
+        viewModel.updateUiState("")
+
+        //  assertEquals(viewModel.tagError.value, R.string.tag_error_empty)
```

Comment:
I think this is not on purpose, right?

Reply:
No, hehe. Fixed in edacb59763f915297662d0e7afc528c58a63511c.